### DISCUSSION TITLE
Command line usage

### DISCUSSION THREAD

**Discussion by Xin74** [jovibor/Pepper#6] [Ideas]
Hello,
Your 'Pepper' program is really useful, many thanks for that!!
My question is if it can be used from the command line?
I am looking for a way to search for and replace text strings in a file appended to an executable.

Many thanks 

Anthony

**Comment by jovibor** [comment]
Hi,

Pepper is a totally GUI app, it was never meant to be used as **CL**.
Search and replace for text strings in the file for me sounds more like a work for a hex editor.🍺🍺
